### PR TITLE
Added French

### PR DESCRIPTION
**Pull request by Leoriem-code**
As the name say, I wrote some explanation in french

### CODE REVIEW

**Review by vbrazo — DISMISSED** [review]

File: fr/math/Trouver_Minimum.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+Dans notre exemple, on va prendre la liste $16,6,2,-14,43,-28,-35,20,-23,25,9,38,17,43,-44,-16,-16,39,\\-4,4,2,-42,11,-13,30,-47,7,-10$
+
+### Exemple en vidéo
+
+![Trouver Minimum](../Exemples/.gif/FindMin.gif)
+
+## Implementation
+
+## Vidéo
```
Comment (Panquesito7) [inline fr/math/Trouver_Minimum.md:39]: Missing implementation and video links. If no implementation is available, feel free to remove that section. 🙂

File: fr/math/Trouver_Maximum.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Dans notre exemple, on va prendre la liste $-48,0,8,22,0,31,-38,-15,43,14,22,21,44,15,-36,-4,-17, -47,45\\,-5,35,-26,-48,-17,-14,47,-39,41$
+
+### Exemple en vidéo
+
+![Trouver Maximum](../Exemples/.gif/FindMax.gif)
+
+## Implementation
+
+## Vidéo
```
Comment (Panquesito7) [inline fr/math/Trouver_Maximum.md:39]: Same here.

File: fr/math/Suite_de_Fibonacci_récursive.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+### Conclusion
+
+On a donc $F(4) = 3$
+
+## Implementation
+
+* [Python](https://github.com/TheAlgorithms/Python/blob/master/maths/fibonacci_sequence_recursion.py)
+
+## Vidéo
```
Comment (Panquesito7) [inline fr/math/Suite_de_Fibonacci_récursive.md:53]: Missing video link.

File: fr/math/Suite_de_Fibonacci.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+### Conclusion
+
+On a donc $F(6) = 8$
+
+## Implementation
+
+* [Python](https://github.com/TheAlgorithms/Python/blob/master/maths/fibonacci.py)
+
+## Vidéo
```
Comment (Panquesito7) [inline fr/math/Suite_de_Fibonacci.md:52]: Missing video link.

File: fr/math/Determinant.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+![Exemple Determinant](../Exemples/.gif/Determinant.gif)
+
+### Conclusion
+
+Le déterminant de $\begin{bmatrix}1&2&3\\4&5&6\\7&8&9\end{bmatrix}$ est $0$.
+
+## Implementation
+
+## Vidéo
```
Comment (Panquesito7) [inline fr/math/Determinant.md:42]: Missing video link.

File: fr/filtres_audio/filtre_butterworth.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+## Étapes
+
+* On commence par calculer les différents coefficients pour le filtre.
+* On applique ces coefficients à un filtre IIR.
+
+## Implementation
+
+* [Python](https://github.com/TheAlgorithms/Python/audio_filters/butterworth_filter.py)
+
+## Vidéo
```
Comment (Panquesito7) [inline fr/filtres_audio/filtre_butterworth.md:35]: Missing video link.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Great work! 😄👍

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Comment by Panquesito7** [comment]
@appgurueu, would you like to review this PR? Thanks. 🙂

**Review by vbrazo — APPROVED** [review]

**Pull request merged**